sources/packages/forms/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN ln -sf $NPM_PACKAGES/node_modules node_modules
 ENV DEBUG=""
 
 RUN set -x \
-  && chmod -R 777 /app/
+  && chmod -R 770 /app/
```

Comment:
Asking for clarification?

What is the standard that we are trying to comply with here (by changing permission)?

Sonar? or any security vulnerability issue from scans previously?

Reply:
We are trying to be less permissive as possible and still have the form.io server running.
![image](https://github.com/bcgov/SIMS/assets/61259237/c6ec757f-01cb-4b11-acfb-f531b097ff45)

Reply:
Security vulnerability to restrict others to have any permission other than the owner and group.